content/en/docs/setup/version-skew-policy.md

```diff
@@ -0,0 +1,112 @@
+---
+reviewers:
+- sig-api-machinery
+- sig-architecture
+- sig-cli
+- sig-cluster-lifecycle
+- sig-node
+- sig-release
+title: Kubernetes Version and Version Skew Support Policy
+content_template: templates/concept
+weight: 70
+---
+
+{{% capture overview %}}
+This document describes the maximum version skew supported between various Kubernetes components.
+Specific cluster deployment tools may place additional restrictions on version skew.
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Supported versions
+
+The Kubernetes project maintains release branches for the most recent three minor releases,
+and expects users to be running those versions in production.
+
+Applicable bug and security fixes may be backported to those three release branches, depending on severity and feasibility.
+Patch releases are cut from those branches at a regular cadence,
+or on an as-needed basis as determined by the [patch release manager](https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/patch-release-manager/README.md#release-timing).
+
+With minor releases happening approximately every three months,
+that means each minor release is maintained for approximately nine months.
+
+## Supported version skew
+
+### `kube-apiserver`
+
+In highly available (HA) deployments, the newest and oldest `kube-apiserver` must be within one minor version.
+
+Example:
+
+* newest `kube-apiserver` is at `1.x`
+* other `kube-apiserver` components are supported at `1.x` and `1.x-1`
+
+### `kubelet`
+
+`kubelet` must not be newer than `kube-apiserver`, and may be up to two minor versions older.
+
+Example:
+
+* `kube-apiserver` is at `1.x`
+* `kubelet` is supported at `1.x`, `1.x-1`, and `1.x-2`
+
+Note that if `kube-apiserver` skew exists in an HA control plane, this narrows the allowed `kubelet` versions.
+
+Example:
+
+* `kube-apiserver` components are at `1.x` and `1.x-1`
+* `kubelet` is supported at `1.x-1`, and `1.x-2` (`1.x` is not supported because that would be newer than the `kube-apiserver` at version `1.x-1`)
+
+### `kube-controller-manager` and `kube-scheduler`
+
+`kube-controller-manager` and `kube-scheduler` must not be newer than `kube-apiserver`. They are expected to match the `kube-apiserver` minor version, but may be one minor version older during upgrades.
+
+Example:
+
+* `kube-apiserver` is at `1.x`
+* `kube-controller-manager` and `kube-scheduler` are supported at `1.x` and `1.x-1`
+
+Note that if `kube-apiserver` skew exists in an HA control plane, this narrows the allowed `kube-controller-manager` versions.
```

Comment:
i guess we can distinguish between a multi-control-plane cluster and an HA cluster, but this is fine as is.

Reply:
clarified language to `multi-server cluster`